docs/usage/help.rst.inc

```diff
@@ -149,7 +149,7 @@ Examples::
 
     A more general and easier to use way to define filename matching patterns exists
     with the experimental ``--pattern`` and ``--patterns-from`` options. Using these, you
-    may specify the backup roots (starting points) and patterns for inclusion/exclusion.
+    may specify the backup roots (starting points), pattern files, and patterns for inclusion/exclusion.
```

Comment:
this paragraph is talking about the patterns specifically (not the command line in general), so we are already "in" the pattern file, so you can't specify a pattern file there.

so I'ld suggest to just remove this change.